src/ChatPage.cpp

```diff
@@ -450,7 +450,7 @@ ChatPage::ChatPage(QSharedPointer<UserSettings> userSettings, QWidget *parent)
           this, &ChatPage::updateGroupsInfo, communitiesList_, &CommunitiesList::setCommunities);
 
         connect(this, &ChatPage::leftRoom, this, &ChatPage::removeRoom);
-        connect(this, &ChatPage::notificationsRetrieved, this, &ChatPage::sendDesktopNotifications);
+        connect(this, &ChatPage::notificationsRetrieved, this, &ChatPage::sendNotifications);
```

Comment:
Not sure, if the rename makes that much sense, since I don't think the alerts work on any platform but the desktop, but I always felt like the Desktop was a bit redundant anyway, so I think the rename makes sense.

Reply:
Well it's a notification on the user's desktop, not necessarily for desktop platforms. That's the way I interpreted it at least. Maybe "popupNotification" would be a more descriptive name?
Of course on Linux it's not defined what will eventually happen with that DBus notification, but that's what regular users would be used to I think.

Reply:
Mhm, yeah, sounds reasonable.